web/garbage.go

```
@@ -85,18 +96,130 @@ func garbage(c *gin.Context) {
+		dlChunkSize = cfg.Speedtest.DownDataChunkSize * 1024 * 1024
+	}
+
+	dlChunks = defaultChunks
+	if cfg.Speedtest.DownDataChunkCount > 0 {
+		dlChunks = cfg.Speedtest.DownDataChunkCount
+	}
+
+	fmt.Printf("Initializing random data seed (%d bytes)...\n", dlChunkSize)
+	// 使用 crypto/rand 生成高质量的随机种子
+	seed := make([]byte, dlChunkSize)
+	_, err = rand.Read(seed)
```

golangci-lint (1.64.8)

SA1019: rand.Read has been deprecated since Go 1.20 because it shouldn't be used: For almost all use cases, [crypto/rand.Read] is more appropriate. If a deterministic source is required, use [math/rand/v2.ChaCha8.Read]. 

(staticcheck)